### ISSUE TITLE
Customised DP for account.

### ISSUE THREAD

**Issue by Rinhak** [recloudstream/cloudstream#1022]
### Describe your suggested feature

Cloudstream should have an option to put dp in the account selection at the home screen. Like Prime or Netflix.
![IMG_20240402_162629](https://github.com/recloudstream/cloudstream/assets/99042103/a329ef59-571a-40f4-9edc-52860d1d470e)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by mizu-048** [comment]
work in progress along with account sync.

**Comment by IndusAryan** [comment]
will see probably after #1001

**Comment by Rinhak** [comment]
Hello, can we get this update if possible please? Wanted a customised DP option in Cloudstream's home screen. The anilist dp doesn't appear so if we had a different option.

**Comment by Rinhak** [comment]
> will see `Probably` after #1001

Hello, can we get this update if possible please? Wanted a customised DP option in Cloudstream's home screen. The anilist dp doesn't appear so if we had a different option.

**Comment by Rinhak** [comment]
Will this be taken forward?
